src/index.js

```diff
@@ -6,10 +6,12 @@ const logIt = require('./helpers/logger.js');
 const { stdNum } = require('./helpers/math.js');
 const { twilioActivated, notifyUserViaText } = require('./notifier');
 const { FIFTEEN_MINS_MS } = require('./helpers/constants.js');
```

Comment:
Can you remove this const and also delete all time related constants from there?

If there is nothing left in the file it can be deleted too.